src/main/java/frc/robot/RobotContainer.java

```diff
@@ -30,21 +32,21 @@ public RobotContainer() {
     }
 
     private void configureDefaultCommands() {
-
     }
 
     private void configureButtonBindings() {
-
+        a.whenPressed(new InstantCommand(() -> hood.setSolenoid(Hood.Mode.ShortDistance), hood));
+        b.whenPressed(new InstantCommand(() -> hood.setSolenoid(Hood.Mode.LongDistance), hood));
     }
 
 
     /**
-    * Use this to pass the autonomous command to the main {@link Robot} class.
-    *
-    * @return the command to run in autonomous
-    */
+     * Use this to pass the autonomous command to the main {@link Robot} class.
+     *
+     * @return the command to run in autonomous
+     */
     public Command getAutonomousCommand() {
-      return null;
+        return null;
```

Comment:
This change is made in too many branches. Instead, create a small PR for formatting fixes. Then, make sure you format your code before committing it. [This can be done easily with IntelliJ IDEA's excellent Git integration](https://www.jetbrains.com/help/idea/commit-and-push-changes.html#commit) and will prevent redundant diffs.

Reply:
I'm having trouble deleting the extra spaces.

Reply:
First, if you're having trouble – why did you resolve the conversation?
Second, what is the problem?

Reply:
I figured it out. Turns out I just needed to highlight the spaces 😅.